rites/roman1969/src/locales/la.ts

```diff
@@ -8,7 +8,7 @@ export const locale: Locale = {
       season: 'Adventus',
       weekday: '$t(weekdays:{{dow}}, capitalize), hebdomada $t(ordinals:{{week}}, { "context": "feminine" }) Adventus',
       sunday: 'Dominica $t(ordinals:{{week}}, { "context": "feminine" }) Adventus',
-      privileged_weekday: 'Dia {{day}} $t(months:11)',
+      privileged_weekday: 'Die {{day}} $t(months:11)',
```

Comment:
A [project of dogitiling the Liturgy of the Hours](https://lh.kbs.sk/cgi-bin/l.cgi?qt=pdt&d=17&m=12&r=2025&j=la&o2=29368) (into Latin, among other languages) uses `Feria quarta, Tempus Adventus`.